eza.plugin.zsh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# Exit if the 'eza' command could not be found
+if ! (( $+commands[eza] )); then
+    echo "ERROR: 'eza' command not found"
+    return
+fi
+
+# Create alias override commands using 'eza'
+alias ls='eza --group-directories-first --icons'
+
+# Use the --git flag if the installed version of eza supports git
+# Related to https://github.com/ogham/exa/issues/978
+if exa --version | grep -q '+git';
```

Comment:
Maybe you forgot to change the executable name also in this line. Even if te eza installation provides a symlink to exa for backword compatibility change this to eza is a matte of consitency